README.md

```diff
@@ -156,6 +156,16 @@ To start tracking the user's location in the background, call:
 Radar.startTracking();
 ```
 
+Optionally, you can configure advanced tracking options. See the [iOS background tracking documentation](https://radar.io/documentation/sdk#ios-background) and [Android background tracking documentation](https://radar.io/documentation/sdk#android-background) for descriptions of these options.
+
+```javascript
+Radar.startTracking({
+  priority: 'responsiveness', // Android only, use 'efficiency' to avoid Android vitals bad behavior thresholds
+  sync: 'possibleStateChanges', // use 'replayOff' to disable offline replay ('replayStopped' recommended)
+  offline: 'replayStopped' // use 'all' to sync all location updates ('possibleStateChanges' recommended)
```

Comment:
the comments are swapped between the above two lines

Reply:
Fixed